src/SFA.DAS.EmployerIncentives.Web/ViewModels/Applications/ViewApplicationsViewModel.cs

```diff
@@ -69,7 +67,7 @@ public string ToggleSortOrder(string fieldName)
             return "asc";
         }
 
-        public BankDetailsStatus BankDetailsStatus { get; set; }
+        public bool ShowBankDetailsCalltoAction { get; set; }
```

Comment:
This should be something like ShowBankDetailsRequiredCallToAction. There's different content shown for bank status of In Progress

Reply:
OK.  Renamed to ShowAddBankDetailsCalltoAction